source/declarations.tex

```diff
@@ -2040,10 +2040,15 @@
 If the \grammarterm{enum-key} is followed by a
```

Comment:
Please add "in an enum-head or opaque-enum-declaration" after "enum-key". I think we may have duplicated this previously as a misguided and unclear attempt to address both cases separately.

Reply:
Hmm, could we refactor opaque-enum-declaration to use enum-head-name, and reference that here? That would avoid confusion over whether this applies if there is an intervening attribute-specifier-seq.